tensorflow/lite/delegates/webnn/webnn_delegate.cc

```diff
@@ -1762,7 +1762,7 @@ class Subgraph {
       std::unordered_map<int, emscripten::val>& webnn_operands,
       const bool detect_supported_op) {
     TF_LITE_ENSURE_STATUS(
-        CheckNumInputsAndOutputs(logging_context, node, 2, 4, 1, node_index));
+        CheckNumInputsAndOutputs(logging_context, node, 1, 4, 1, node_index));
```

Comment:
Thanks for this fix. Better to have its own commit.

Reply:
Done via https://github.com/huningxin/tensorflow/pull/41.